radix-engine/src/system/system_modules/costing/fee_table.rs

```diff
@@ -381,6 +381,40 @@ impl FeeTable {
         500 + Self::data_processing_cost(size)
     }
 
+    #[inline]
+    pub fn bls12381_v1_verify_cost(&self, size: usize) -> u32 {
+        // Based on benchmark `bench_bls_verify`
```

Comment:
Not sure if it is fine to rely on benchmark only.

Reply:
Now relying on instructions count - measured in this [QEMU costing job](https://github.com/radixdlt/radixdlt-scrypto/actions/runs/7410771396)